packages/discovery-provider/src/models/users/chat_blocked_users.py

```diff
@@ -0,0 +1,12 @@
+from sqlalchemy import Column, DateTime, Integer
+
+from src.models.base import Base
+from src.models.model_utils import RepresentableMixin
+
+
+class ChatBlockedUser(Base, RepresentableMixin):
+    __tablename__ = "chat_blocked_users"
+
+    blocker_user_id = Column(Integer, primary_key=True, nullable=False)
+    blockee_user_id = Column(Integer, primary_key=True, nullable=False, index=True)
+    created_at = Column(DateTime, nullable=False)
```

Comment:
Is this a model of the db that comms contributes to?

Reply:
yup

Reply:
nice, so then the current way to test blocking is just via the DMs screen?

Reply:
yea but you can also block on a user's profile. I didn't see designs for this yet but maybe i missed it